docs/content/api.md

```diff
@@ -0,0 +1,35 @@
+# API
+
+Maesh includes a built-in API that can be used for debugging purposes.
+This can be useful when Maesh is not working as intended.
+The API is accessed via the controller pod, and for security reasons is not exposed via service.
+The API can be accessed by making a `GET` request to `http://<control pod IP>:9000` combined with one of the following paths:
+
+## `/api/configuration/current`
+
+This endpoint provides raw json of the current configuration built by the controller.
+
+!!! Note
+    This may change each request, as it is a live data structure.
+
+## `/api/status/nodes`
+
+This endpoint provides a json array containing some details about the readiness of the Maesh nodes visible by the controller
+This endpoint will still return a 200 if there are no visible nodes.
+
+## `/api/status/node/{node}/configuration`
```

Comment:
Not convinced by this change, but I would be nice if we had a hint about using the pod name directly.

```suggestion
## `/api/status/node/{maesh-pod-name}/configuration`
```